packages/remote/lib/adapter.js

```diff
@@ -73,17 +73,31 @@ class FuryRemoteAdapter {
   }
 
   parse({
-    source, generateSourceMap, namespace, mediaType,
+    source, generateSourceMap, generateMessageBody, generateMessageBodySchema, namespace, mediaType,
   }) {
     const inputMediaType = mediaType || detectMediaType(source, defaultInputMediaType);
 
+    if (generateMessageBody === undefined) {
+      this.generateMessageBody = true;
+    } else {
+      this.generateMessageBody = generateMessageBody;
+    }
+
+    if (generateMessageBodySchema === undefined) {
+      this.generateMessageBodySchema = true;
+    } else {
+      this.generateMessageBodySchema = generateMessageBodySchema;
+    }
+
```

Comment:
I don' tthink this logic is needed, we can propergate it below directly without transformation and handle the defaults on the server side. If `generateMessageBody` is undefined, then we can prevent it being sent to the API as a parameter and let the API handle the default.

Below could become:

```
params: {
  generateSourceMap,
  generateMessageBody,
  generateMessageBodySchema,
}
```

At least I think that would work, perhaps I am mis understanding how axios is working.